src/js/containers/AppList.js

```diff
@@ -36,13 +36,13 @@ const mapStateToProps = (state) => {
             name: name,
             devicename: devicename,
             image: icon,
-            link: '/' + link,
+            link: undefined,
```

Comment:
What was the purpose of this change?

Reply:
This prevents the HMI from moving to a blank media template immediately when an app is clicked on.